### PR TITLE
Lesser chems

### PR DESCRIPTION
**Pull request by SonixApache**
One day I ran into a barkeep that wanted to make some extra cash selling drugs. But there's not a lotta interesting lesser-effect drugs to take. That and some other recent experiences inspired me to make some stuff.

I've got some more ideas in the works, but these are all good to go.

TODO:
 - [x] update wiki (post-merge)

🆑 
 - rscadd: Added Mannitol, a commonly requested medicine
 - rscadd: Added Dietine, a weight loss supplement.
 - rscadd: Added Gator Mix, a sugary nutrient supplement.
 - rscadd: Added Blisterol, a brute-healing drug that causes burns.
 - rscadd: Added Magica, a bitter alcoholic drink made from mixing two magical bottles.
 - rscadd: Added Magica Deluxe, a special drink made by mixing Magica with some Karmotrine.

### CODE REVIEW

**Comment by jknpj** [comment]
>mannitol

howaiTo piGgo go homu

**Comment by ghost** [comment]
> 
> 
> > mannitol
> 
> howaiTo piGgo go homu

https://en.wikipedia.org/wiki/Mannitol

**Comment by PrimeDSS13** [comment]
mannitol...

**Comment by BomberBro** [comment]
Dislike mannitol being added, especially when it actually works as a brian damag medicine.

**Comment by SonixApache** [comment]
Would you :ok: it if it gravitated brain damage to 10 instead of 1?

**Comment by BomberBro** [comment]
I don't really see the point of mannitol when you can make actual brain damage chem just as easily, I also have the issue with dietine since lipo is arguably easier to make. I really like the idea of chems with weird and off recipes that you'd have to go out of your way for or could make in a ghetto chem lab, like pepto or what you've done with the burn chem, but it's hard to justify making these lower chems when you can make the better version for less effort and with greater effect.

**Comment by SonixApache** [comment]
i'll make some changes in a bit, agree with you on that one

**Comment by SonixApache** [comment]
Modified the recipes, check it out.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -3412,6 +3430,33 @@
...
+	density = 1.8
+	specheatcap = 0.66
+
+/datum/reagent/mannitol/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	if(M.getBrainLoss() > 10 && prob(40))
+		M.adjustBrainLoss(-1 * REM)
```
Comment (jknpj) [inline code/modules/reagents/Chemistry-Reagents.dm]: Make it work as a weak painkiller + brain damage inducer instead of enabling TG refugees.

Comment (SonixApache) [reply code/modules/reagents/Chemistry-Reagents.dm]: fixed

**Comment by SonixApache** [comment]
Added a karmotrine drink that to my eye is full of shitcode. But it works.

**Comment by jknpj** [comment]
update wiki

**Pull request merged**